etcdctl/ctlv3/command/get_command.go

```diff
@@ -47,6 +48,7 @@ func NewGetCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&getPrefix, "prefix", false, "get keys with matching prefix")
 	cmd.Flags().BoolVar(&getFromKey, "from-key", false, "get keys that are greater than or equal to the given key")
 	cmd.Flags().Int64Var(&getRev, "rev", 0, "specify the kv revision")
+	cmd.Flags().BoolVar(&getKeysOnly, "keys-only", false, "get only the keys (valid only with --prefix)")
```

Comment:
Yes, it would work without --prefix. I thought it would make more sense for --prefix. But you are right that it would be useful if somebody wants to just get the details of the key without the value. I'll make the necessary changes.